roles/kubernetes-apps/persistent_volumes/aws-ebs-csi/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - alijahnas
```

Comment:
I Assume you meant to add yourself in the "reviewers" section?

Reply:
I thought about adding myself as a reviewers but @Miouge1 in a comment above said to add as an approver.

Reply:
my mistake, marking this as closed